src/Hintman/Suggestion/TrailingSpaces.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE NamedFieldPuns #-}
```

Comment:
I feel that usage of both `RecordWildCards` and `NamedFieldPuns` is redundant. With our naming scheme we shouldn't have problems with using only `RecordWildCards`